src/haz3lweb/view/Page.re

```diff
@@ -75,6 +80,15 @@ module Update = {
         action: Globals.Update.t,
         model: Model.t,
       ) => {
+    /* A function passed down to trigger an update within
+       assistant which checks for the insertion of '??' or '?a' */
+    let send_insertion_info = (~char, ~editor) => {
```

Comment:
not sure specifically but name should probably advertise this is for the assistant.